pkg/api/serialization_test.go

```diff
@@ -415,9 +416,15 @@ func TestSpecificKind(t *testing.T) {
 	}
 }
 
+// Keep this in sync with the respective upstream set
+var nonInternalRoundTrippableTypes = sets.NewString("List", "ListOptions")
+
 func TestTypes(t *testing.T) {
 	for kind, reflectType := range api.Scheme.KnownTypes("") {
-		if !strings.Contains(reflectType.PkgPath(), "/origin/") {
+		if !strings.Contains(reflectType.PkgPath(), "/origin/") && reflectType.PkgPath() != "k8s.io/kubernetes/pkg/api" {
```

Comment:
ah. Can we update the name of this test if the intent is to test serialized k8s types?